src/helpers/types.ts

```diff
@@ -88,6 +88,12 @@ export interface PwCommandComposer {
   maxControlLengthClass?: number;
   tags?: string[];
   formatInQuotes?: boolean;
+  possibleValues?: string[] | GeolocationOption[];
+}
+
+export interface GeolocationOption {
```

Comment:
I would name it more generic, eg `DisplayValueOption` so later we can use for other options.

Reply:
Updated the variable name to be more generic.